src/composables/multisigAccounts.ts

```diff
@@ -0,0 +1,294 @@
+    activeMultisigAccountId.value = gaAccountId;
+  }
+
+  async function updateMultisigAccounts() {
+    const sdk = await getSdk();
+
+    let rawMultisigData: IMultisigAccount[] = [];
+    try {
+      await Promise.all(accounts.value.map(async ({ address }) => rawMultisigData.push(
+        ...(await fetchJson(`${activeNetwork.value.multisigBackendUrl}/${address}`)),
+      )));
+    } catch {
+      // TODO: handle failure in multisig loading
+      // eslint-disable-next-line no-console
+      console.log('failed to fetch multisigAccounts');
+    }
+
+    rawMultisigData = uniqBy(rawMultisigData, 'contractId');
+
+    const result = (await Promise.all(
+      rawMultisigData.map(async ({ contractId, ...otherMultisig }) => {
+        const contractInstance = await sdk.getContractInstance({
+          source: multisigContract,
+          contractAddress: contractId,
+        });
+
+        const nonce = (await contractInstance.methods.get_nonce()).decodedResult;
+        const signers = (await contractInstance.methods.get_signers()).decodedResult;
+        const version = (await contractInstance.methods.get_version()).decodedResult;
+
+        const consensus = (
+          await contractInstance.methods.get_consensus_info()
+        ).decodedResult as any;
+
+        if (consensus.tx_hash) {
+          consensus.tx_hash = Buffer.from(consensus.tx_hash).toString('hex');
+        }
+        consensus.expiration_height = Number(consensus.expiration_height);
+        consensus.confirmations_required = Number(consensus.confirmations_required);
+
+        return {
+          ...camelcaseKeysDeep(consensus),
+          ...otherMultisig,
+          nonce: Number(nonce),
+          signers,
+          version,
+          contractId,
+          balance: (await sdk.balance(otherMultisig.gaAccountId)) || 0,
+          address: otherMultisig.gaAccountId,
+        };
+      }),
+    ))
+      .sort((a, b) => b.balance - a.balance)
+      .sort((a, b) => b.confirmedBy.length - a.confirmedBy.length)
+      .sort((a) => (
+        a.txHash
+        && a.signers.some((signer: string) => (
+          accounts.value.map(({ address }) => address).includes(signer)
+          && !a.confirmedBy.includes(signer)
+        ))
+      ) && -1);
+
+    if (!activeMultisigAccountId.value) {
+      setActiveGaAccountId(result[0]?.gaAccountId);
+    }
+
+    multisigAccounts.value = result;
+    storeMultisigAccounts(result, activeNetwork.value.networkId);
+  }
+
+  async function buildSpendTx(
+    senderId: string,
+    recipientId: string,
+    amount: number,
+  ) {
+    const sdk = await getSdk();
+
+    return sdk.spendTx({
+      senderId,
+      recipientId,
+      amount,
+    });
+  }
+
+  async function postSpendTx(tx: string, txHash: string) {
+    return postJson(`${activeNetwork.value.multisigBackendUrl}/tx`, { body: { hash: txHash, tx } });
+  }
+
+  async function proposeTx(spendTx: string, contractId: string) {
```

Comment:
This and few other methods are strictly related to the multisig transactions - nothing to do with multisig accounts. So maybe creating separate composable for the `multisigTransactions` would be a good idea.